Dockerfile.kubetest2

```diff
@@ -56,3 +56,5 @@ RUN cp kubernetes-version.txt /info/
 RUN mv kubernetes/*/bin/* /bin/
 RUN rm -rf /workdir
 COPY --from=builder /usr/local/go/bin/* /bin/
+RUN mkdir -p /etc/ssl/certs/
+COPY certs.pem /etc/ssl/certs/certs.pem
```

Comment:
We should either use an `ARG` for this or just mount the certs in with a volume at runtime